cmd/juju/machine/upgrademachine_test.go

```diff
@@ -69,12 +70,12 @@ func (s *UpgradeMachineSuite) SetUpTest(c *gc.C) {
 	s.completeExpectation = &upgradeMachineCompleteExpectation{gomock.Any()}
 
 	// TODO: remove this patch once we removed all the old series from tests in current package.
```

Comment:
Should this todo still be here now that we have removed series?

Reply:
This whole patch can be dropped actually